tools/generate_json_rpc_docs.py

```diff
@@ -211,6 +211,17 @@ def to_markdown(self):
 - A **response** from Jamulus to the consumer.
 - A **notification** from Jamulus to the consumer.
 
+**Tip:** On Linux, you can connect to a JSON-RPC server using the `nc` CLI tool. On Windows, you can download and use the `ncat` CLI tool.  This snippet uses [jayson](https://www.npmjs.com/package/jayson) to connect using Node.js:
```

Comment:
```suggestion
## Connecting to the JSON-RPC server

On **Linux**, you can use the `nc` CLI tool. 

On **Windows**, you can download and use the `ncat` CLI tool. 

To control the JSON-RPC server programmatically, you can use a package like [jayson](https://www.npmjs.com/package/jayson) for Node.js line this:
```

I'd give a link to ncat if it isn't installed by default. Maybe even give examples for the nc commands.